nginx.conf

```diff
@@ -148,10 +148,11 @@
     }
 
     # Swagger UI
-    rewrite ^/swagger$ $1/swagger/;
-    rewrite ^/swagger.json$ /swagger/swagger.json;
+    rewrite ^/swagger/swagger.json$ /swagger.json permanent;
```

Comment:
Not sure we need this, maybe only for backward compatibility, since some may have been looking for a json output via /swagger/swagger.json.